### PR TITLE
update website contributionpage

### LINKED ISSUE #135: Docs: Ksctl Contributions docs should be improve
# Why do we need it?
- On Ksctl Contributions docs should have steps improvement on locally setup. for new contributor
# What it will bring?
- We should add things like :
- Go to the Ksctl GitHub repo <LINK>.
- Clone it by doing Git clone ... 
- also add the build process.
## Additional context

Check Contribution's guidelines

### PR DESCRIPTION
**Pull request by Rahul-Kumar-prog**
# Tasks description : modify contribution page on website.

- [ ] Closes #135 

# Solution :heavy_check_mark:

# ScreenShorts: 
Screenshot 2023-08-20 at 7 09 14 PM
Screenshot 2023-08-20 at 7 09 44 PM

- [x] Ran Unit Test locally
- [x] Tested the website locally
- [x] Checked Contribution's guidelines

### CODE REVIEW

File: website/docs/contributions.md [hunk 1]
```diff
@@ -47,32 +47,53 @@ what? why? How?
...
-
-Unit tests you can do my
+# Development
+First you have to fork the ksctl repository. [fork](https://github.com/kubesimplify/ksctl/fork)
 ```bash
-make test
+cd <path> # to you directory where you want to clone ksctl
+mkdir <directory name> # create a directory
+cd <directory name> # go inside the directory
+git clone git@github.com:${YOUR_GITHUB_USERNAME}/ksctl.git # clone you fork repository
```
Comment (dipankardas011) [inline website/docs/contributions.md:56]: Use HTTPS based clone method as most of them don't use SSH

File: website/docs/contributions.md [hunk 2]
```diff
@@ -47,32 +47,53 @@ what? why? How?
...
+cd ksctl # go inside the ksctl directory
+git remote add upstream https://github.com/kubesimplify/ksctl.git # set upstream
+git remote set-url --push upstream no_push # no push to upstream
 ```
 
-Trying out latest code by installing
+# Making Build
```
Comment (dipankardas011) [inline website/docs/contributions.md:62]: Building from source

File: website/docs/contributions.md [hunk 3]
```diff
@@ -47,32 +47,53 @@ what? why? How?
...
 .\builder.ps1 # for windows
 ```
+## for website
+```bash
+cd website # to go inside the directory
+```
+### Install Dependencies
+```bash
+npm install # install npm  
+npm install --global yarn # install yarn
```
Comment (dipankardas011) [inline website/docs/contributions.md:82]:
```suggestion
npm start
# or
yarn start
```

**Review by dipankardas011 — CHANGES_REQUESTED** [review]
Some minor changes needed

File: website/docs/contributions.md [hunk 4]
```diff
@@ -47,32 +47,53 @@ what? why? How?
...
+### Mac OS
+```bash
 make install_macos # for macos
-
+```
+### Windows
+```bash
 .\builder.ps1 # for windows
 ```
+## for website
```
Comment (dipankardas011) [inline website/docs/contributions.md:75]:
```suggestion
## Website
```

File: website/docs/contributions.md (hunk 3 above)
Comment (Rahul-Kumar-prog) [reply website/docs/contributions.md:82]: @dipankardas011  are you sure npm start works here?

**Comment by Rahul-Kumar-prog** [comment]
@dipankardas011 sorry for the late. have a look

File: go.sum [hunk 5]
```diff
@@ -134,6 +134,7 @@ github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0 h1:p104kn46Q8Wd
 github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0/go.mod h1:fyg7847qk6SyHyPtNmDHnmrv/HOrqktSC+C9fM+CJOE=
 github.com/gogo/protobuf v1.3.2 h1:Ov1cvc58UF3b5XjBnZv7+opcTcQFZebYjWzi34vdm4Q=
 github.com/gogo/protobuf v1.3.2/go.mod h1:P1XiOD3dCwIKUDQYPy72D8LYyHL2YPYrpS2s69NZV8Q=
+github.com/golang-jwt/jwt v3.2.1+incompatible/go.mod h1:8pz2t5EyA70fFQQSrl6XZXzqecmYZeUEB8OUGHkxJ+I=
```
Comment (dipankardas011) [inline go.sum:137]: not sure why it changed

File: website/docs/contributions.md [hunk 6]
```diff
@@ -72,14 +72,14 @@ make install_macos # for macos
...
-## for website
+## Website
 ```bash
 cd website # to go inside the directory
 ```
 ### Install Dependencies
 ```bash
 npm install # install npm  
-npm install --global yarn # install yarn
+npm start # install yarn
```
Comment (dipankardas011) [inline website/docs/contributions.md:82]: you can add both the methods to try
yarn start
npm start

**Review by dipankardas011 — CHANGES_REQUESTED** [review]
some minor ones

**Comment by dipankardas011** [comment]
@Rahul-Kumar-prog  are you still working on this issue??

**Comment by Rahul-Kumar-prog** [comment]
@dipankardas011  NO, I have done what I suggested in the issue. If there are any other improvements please let me know.

**Review by dipankardas011 — APPROVED** [review]
LGTM

**Pull request merged**